src/rebar3.erl

```diff
@@ -320,7 +322,6 @@ ensure_running(App, Caller) ->
     end.
 
 state_from_global_config(Config, GlobalConfigFile) ->
-    rebar_utils:set_httpc_options(),
```

Comment:
Had to move it to rebar3:init_config. GlobalConfigFile is missing in my case and thus httpc_options were not setting at all.